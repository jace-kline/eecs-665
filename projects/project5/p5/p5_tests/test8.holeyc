int f(int a) {
    return (a > 0);
}

int main(int argc, charptr argv) {
    bool b;
    b = true;
    b = b && f(b);
    b = b && f(1,2);
    return 0;
}