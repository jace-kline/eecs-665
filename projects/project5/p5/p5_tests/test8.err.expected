FATAL [2,15]: Bad return value
FATAL [8,16]: Type of actual does not match type of formal
FATAL [8,14]: Logical operator applied to non-bool operand
FATAL [9,14]: Function call with wrong number of args
FATAL [9,14]: Logical operator applied to non-bool operand
Type Analysis Failed